src/Pages/WhyVacateEmployment.jsx

```diff
@@ -1,9 +1,151 @@
+  },
+  {
+    summary: "Oil and gas professionals",
+    details:
+      "The oil and gas industry offers endless opportunities for job seekers at any level.",
+  },
+  {
+    summary: "App and software development companies",
+    details:
+      "Mobile applications and software developers are some of the highly sought-after professionals in the United States. Every business that takes online marketing seriously has mobile apps. Technology companies that offer mobile app development and software services are always on the lookout for the best talents in the industry. If you have what it takes, your felony will not always come in your way of getting jobs like this one.",
+  },
+  {
+    summary: "Computer repair technician jobs",
+    details:
+      "Computer repair technicians are estimated to grow by more than 8.2 percent over the next decades. In 2016 alone, the number of computer technicians was 69,030 in California, 43,800 in New York, 21,700 and 6,420 in North Carolina and South Carolina, respectively, as well as over 500,000 more all over the United States.",
+  },
+  {
+    summary: "Retail companies",
+    details:
+      "Retail companies will hire people with felonies, but are less likely to hire people with a records of theft or violence.",
+  },
+  {
+    summary: "Transport companies",
+    details:
+      "People with felonies can find job opportunities in the transport industry as drivers, carriers, and delivery guys.",
+  },
+  {
+    summary: "Food services and grocery stores",
+    details: "Missing",
+  },
+];
 
 const WhyVacateEmployment = () => {
+  const classes = WhyVacateEmploymentStyles();
   return (
-    <Container>
-        <Typography variant="h2">EMPLOYMENT</Typography>
+    <Container maxWidth="lg">
+      <Grid container alignItems="flex-start">
+        <Typography className={classes.title} variant="h2">
+          Employment barriers
+        </Typography>
+        <Typography className={classes.contentText} variant="body1">
+          Simply having a conviction does not prevent you from having a job,
+          however, it can be more difficult depending on the type of job and the
+          employer. It is important to not lose hope while facing challenges
+          such as:
+        </Typography>
+        </Grid>
+      <FactGroup fact1={unemployment} fact2={rejected} fact3={education} />
+      <TextWithCTAButton
+        text={
+          "The best way to avoid these obstacles is to expunge your record by vacating your conviction. Learn if you’re eligible with our Misdemeanor Eligibility Calculator."
+        }
+        buttonText={"Check Elegibility"}
+      />
+      <Box className={classes.lightBlueSectionBackground}>
```

Comment:
Same as above, the light blue bg does not span the width of the page

Reply:
should be better now.